spec/dummy/app/controllers/application_controller.rb

```diff
@@ -1,5 +1,41 @@
 require "flex_commerce_api/api_base"
 
 class ApplicationController < ActionController::Base
+  include ShiftCommerce::HttpCaching
+  include ShiftCommerce::PreviewStateManagement
+  include ShiftCommerce::NotFoundRedirects
+  include ShiftCommerce::AssetPush
+  include ShiftCommerce::ResourceUrl
```

Comment:
Convert the dummy application controller into a fully-functional application controller, in order to keep all the tests working.